server/services/config/http.go

```diff
@@ -65,11 +65,17 @@ func (h *http) Fetch(ctx context.Context, forge forge.Forge, user *model.User, r
 
 	status, err := h.client.Send(ctx, net_http.MethodPost, h.endpoint, body, response)
 	if err != nil && status != 204 {
-		return nil, fmt.Errorf("failed to fetch config via http (%d) %w", status, err)
+		return nil, fmt.Errorf("failed to fetch config via http from endpoint %s (status: %d): %w", h.endpoint, status, err)
```

Comment:
Not sure, but maybe this publishes private information. If the endpoint is configured globally, anyone who can see the error - and anyone with pipeline access can see it then - can see the endpoint. Yes, it should be configured correctly to check the signature but this we might have to consider. But I'm also fine with ignoring this issue.